apps/stats/src/views/Stats/Growth.tsx

```
@@ -1,236 +1,131 @@
+            }));
+        case 'mrr': {
+            // TODO: replace the hard-coded 958 once real MRR is available
+            const avgMrrPerMember = paidMembers > 0 ? 958 / paidMembers : 0;
+            return allChartData.map(item => ({
+                ...item,
+                value: item.paid * avgMrrPerMember,
+                formattedValue: `$${(item.paid * avgMrrPerMember).toFixed(0)}`,
+                label: 'MRR'
+            }));
+        }
+        case 'total-members':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)